sql/CMakeLists.txt

```diff
@@ -247,6 +247,14 @@ IF(MSVC AND NOT WITHOUT_DYNAMIC_PLUGINS)
   SET_TARGET_PROPERTIES(mysqld_import_lib PROPERTIES IMPORTED_LOCATION ${MYSQLD_LIB})
 ENDIF()
 
+INCLUDE(numa)
```

Comment:
Do not use LINK_LIBRARIES. if you need to link numa to sql library please do so with TARGET_LINK_LIBRARIES(sql numa). if you need for innodb, ditto.